### PR TITLE
replace deprecated methods to suppress warnings

### PR DESCRIPTION
**Pull request by runningjump**

### CODE REVIEW

File: Assets/Scripts/ArenaManagerScript.cs [hunk 1]
```diff
@@ -41,7 +42,7 @@ public class ArenaManagerScript : MonoBehaviour {
     public AudioClip tickUp;
     public AudioClip tickDown;
 
-    private AudioSource audio;
+    private new AudioSource audio;
```
Comment (nicksantan) [inline Assets/Scripts/ArenaManagerScript.cs:45]: What's the 'new' about?

File: Assets/Scripts/ProtipManagerScript.cs [hunk 2]
```diff
@@ -18,28 +19,19 @@ public class ProtipManagerScript : MonoBehaviour {
 		Invoke ("StartGame", proTipTime);
 		// make this a common shared function somehow
 		int playersActive = 0;
-		int whichSoloPlayer = 0;
```
Comment (nicksantan) [inline Assets/Scripts/ProtipManagerScript.cs:21]: Interesting... where did this logic go? Was it even being used?

File: Assets/Scripts/ArenaManagerScript.cs (hunk 1 above)
Comment (runningjump) [reply Assets/Scripts/ArenaManagerScript.cs:45]: For all these AudioSource properties I was getting a message about the class overwriting an `audio` property from their parent class. I guess that `new` indicates that you're meaning to override an existing property.

File: Assets/Scripts/ProtipManagerScript.cs (hunk 2 above)
Comment (runningjump) [reply Assets/Scripts/ProtipManagerScript.cs:21]: It wasn't being used! At least that's what the warning said, and I tested it without it successfully.

**Pull request merged**